src/Files/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -385,8 +385,18 @@ private async void FileList_PreviewKeyDown(object sender, KeyRoutedEventArgs e)
                 if (!IsItemSelected && FileList.Items.Count == 1)
                 {
                     FileList.SelectedIndex = 0;
-                    e.Handled = true;
                 }
+
+                // Open slected directory
+                if (!IsRenamingItem && !ParentShellPageInstance.NavToolbarViewModel.IsEditModeEnabled)
+                {
+                    if (IsItemSelected && SelectedItem.PrimaryItemAttribute == StorageItemTypes.Folder)
+                    {
+                        ItemInvoked?.Invoke(new ColumnParam { NavPathParam = (SelectedItem is ShortcutItem sht ? sht.TargetPath : SelectedItem.ItemPath), ListView = FileList }, EventArgs.Empty);
```

Comment:
Is there a way to keep the focus in the current column?

Reply:
I'll check but it's done in baselayout I think.

Reply:
Also I'd add some delay before opening the folder, what if you're trying to go down many items with the keyboard?

Reply:
Focus is expected to stay in the current column even when opening an item with the mouse.